spice/unattended/RHEL-7-spice.ks

```diff
@@ -50,7 +50,7 @@ firstboot --disable
 #
 # Packages.
 #
-%packages --default
+%packages --default --ignoremissing
```

Comment:
Thank you, btw are you sure they were useless? You are not the only ones using this ks, I remember Suren used to use it in his CI and added some dependencies for his internal tests, but he does not work for us anymore and I'm not sure whether this change affects their setup (now it's maintained by @yashkmankad )

Anyway thank you for this fix, I'll think about a way to avoid such problems in the future (either using per-provider-ks-files or just syncing them as I don't see a reason for those files being on multiple places). Either way need to find a way suitable for all of us.